packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -591,23 +654,23 @@ export class StripeWebhookHandler extends StripeHandler {
   /**
    * Send out an email on payment expiration.
    */
-  async sendSubscriptionPaymentExpiredEmail(paymentSource: Stripe.Source) {
-    const sourceDetails = await this.stripeHelper.extractSourceDetailsForEmail(
-      paymentSource
-    );
+  async sendSubscriptionPaymentExpiredEmail(sourceDetails: {
+    uid: string;
+    email: string | null;
+    subscriptions: FormattedSubscriptionForEmail[];
+  }) {
     const { uid } = sourceDetails;
     const account = await this.db.account(uid);
 
-    await this.mailer.sendSubscriptionPaymentExpiredEmail(
+    return this.mailer.sendSubscriptionPaymentExpiredEmail(
       account.emails,
       account,
       {
         acceptLanguage: account.locale,
         ...sourceDetails,
+        email: sourceDetails.email || account.email,
```

Comment:
Why the `account.email` fallback here?

Reply:
Apparently the Stripe.Customer.email could be null, so it's just to cover that scenario.

Reply:
Realistically that wouldn't be possible since we always create customers with an email.  But it also doesn't hurt.